templates/includes/install-prepare-adb.hbs

```diff
@@ -20,7 +20,16 @@
              <input type="button" class="btn btn-primary clipboard-button" value="&#10697;"></input>
             </div>
           </div>
-          <h4>Arch and derivatives</h4>
+          <h4>openSUSE, SLE</h4>
+          <div class="install-code-wrapper" style="justify-content: center;">
+            <div class="install-code-pre-wrapper">
+              <pre class="install-code-pre"><code>sudo zypper install android-tools</code></pre>
+            </div>
+            <div class="clipboard-button-wrapper">
+             <input type="button" class="btn btn-primary clipboard-button" value="&#10697;"></input>
+            </div>
+          </div>
+          <h4>Arch, Manjaro, EndeavourOS, etc.</h4>
```

Comment:
When writing `Arch and derivatives`, i was assuming every Arch user knows. But indeed might be helpful and looks informative to add Manjaro and EndeavourOS.
Tho the EndeavourOS docs advise to use `yay` for newbies needing the hint that they are arch based, hehe.